packages/autorest.python/autorest/codegen/serializers/builder_serializer.py

```diff
@@ -36,6 +36,7 @@
     ParameterListType,
 )
 from .parameter_serializer import ParameterSerializer, PopKwargType
+from ..models.parameter_list import ParameterType
```

Comment:
Since models.__init__.py also defines `ParameterType`: https://github.com/Azure/autorest.python/blob/17208288144487601c9c6b309b3fa99f4445a76e/packages/autorest.python/autorest/codegen/models/__init__.py#L183-L185, I can't expose ParameterType of parameter_list.py in models.__init__.py otherwise it wi ll raise mypy error https://github.com/Azure/autorest.python/blob/17208288144487601c9c6b309b3fa99f4445a76e/packages/autorest.python/autorest/codegen/models/parameter_list.py#L40-L42.
So I have to import it directly from parameter_list.py